source/publishing_data/08_configuring_dataset_export/configuring_dataset_export.rst

```diff
@@ -0,0 +1,26 @@
+Configuring dataset export
+==========================
+
+Users can download any published dataset in a variety of formats, from the Export tab of the front office (see :doc:`Exporting data </exploring_catalog_and_datasets/04_getting_involved/exporting_data>`).
+
+In the OpenDataSoft platform, the configuration of the available exports is done in the Export tab of the edition interface of each dataset.
+
+Specifying a sort for exports
+-----------------------------
+
+By default, the records of a dataset export are ordered randomly. It is possible to configure the sorting of the records, by choosing a sort type for a chosen field.
```

Comment:
I would rather say `By default, the records of a dataset export are not ordered.`
ordered randomly implies that some shuffle is done each time you download the data, which is not the case.